projects/fal/src/fal/logging/isolate.py

```diff
@@ -17,7 +17,8 @@ class IsolateLogPrinter:
     debug: bool
 
     def __init__(self, debug: bool = False) -> None:
-        self.debug = debug
+        # self.debug = debug
+        self.debug = True
```

Comment:
```suggestion
        self.debug = debug
```

no more need, new isolate is already on shark